OrbitGl/PickingManager.h

```diff
@@ -77,9 +81,10 @@ struct PickingId {
     PickingId id = absl::bit_cast<PickingId, uint32_t>(value);
     return id;
   }
-  uint32_t id : 28;
-  PickingType type : 3;
-  BatcherId batcher_id : 1;
+  
+  uint32_t id_ : kIDBitSize;
+  PickingType type_ : kPickingTypeBitSize;
+  BatcherId batcher_id_ : kBatcherIDBitSize;
```

Comment:
static_assert(kIDBitSize + kPickingTypeBitSize + kBatcherIDBitSize == 32)?

Reply:
It's technically caught by absl::bit_cast<PickingId, uint32_t> - but it's a good idea to add it here, that makes it more explicit for everyone. Added that, thanks.